init-tools.sh

```diff
@@ -157,7 +157,7 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     Tools-Override/crossgen.sh $__scriptpath/Tools
 
     echo "Making all .sh files executable under Tools."
-    ls $__scriptpath/Tools/*.sh | xargs chmod +x
+    find $__scriptpath/Tools/ -name "*.sh" -exec chmod +x {} \;
```

Comment:
I've been burned by adding `find` to the buildtools `init-tools.sh`, since some of our build configs don't have it installed. Have you tested this in our build environments?